### ISSUE TITLE
Is there no way to include a "bcc" field?

### ISSUE THREAD

**Issue by screaser** [lookback/meteor-emails#52]
Tried adding a line to the config for one of my emails along the lines of:

```
    from: 'MyCo <team@MyCo.com>',
    to: recipient.emails[0].address,
    bcc: 'developer@email.com',      <<< 
```

... the idea being that I could watch the early emails to see if things were working as expected. But no luck... didn't find anything in the docs either.

Is there a way to add a "bcc" header line, using lookback/meteor-emails?

If so, perhaps add to the Readme page... if not, I guess this is a [low priority?] feature request. Happy to look into creating a PR if desired; I'm not familiar with the underlying Mailer package though.

**Comment by johanbrook** [comment]
Hi @screaser. I'd be happy to see a PR on this, it's quite trivial. Just pass the proper options to `Email.send` (http://docs.meteor.com/#/full/email_send), which is called in here: https://github.com/lookback/meteor-emails/blob/master/lib/mailer.js#L225 :)

**Comment by johanbrook** [comment]
This is published in [0.7.2](https://github.com/lookback/meteor-emails/releases/tag/v0.7.2).

**Comment by screaser** [comment]
Thanks for the heads up! Very sorry I hadn't gotten around to working on a PR yet... I appreciate your help on this!
